sentry-android/src/main/java/io/sentry/android/event/helper/AndroidEventBuilderHelper.java

```diff
@@ -121,7 +121,9 @@ public void helpBuildingEvent(EventBuilder eventBuilder) {
         ActivityManager.MemoryInfo memInfo = getMemInfo(ctx);
         if (memInfo != null) {
             deviceMap.put("free_memory", memInfo.availMem);
-            deviceMap.put("memory_size", memInfo.totalMem);
+            if (Build.VERSION.SDK_INT >= 16) {
```

Comment:
Use `Build.VERSION_CODES.JELLY_BEAN` instead of the plain (magic) number `16` here. Other than that, it looks good!

Reply:
Sure! Thanks :)